test/application/server-commands.test.js

```diff
@@ -0,0 +1,95 @@
+const simpleGit = require("simple-git");
+const git = simpleGit();
+
+const scenarioUpToDate = {
+    modified: [],
+    ahead: 0,
+    behind: 0,
+}
+
+describe('ServerCommands', () => {
+    const scenarioModifiedOutput = {
```

Comment:
Add more dynamic scenario's here and switch simple-git's output before testing each scenario.